service/process.ml

```diff
@@ -24,10 +24,10 @@ let check_status cmd = function
   | status -> Fmt.failwith "%a %a" pp_cmd cmd pp_status status
 
 let exec cmd =
-  let proc = Lwt_process.open_process_none cmd in
-  proc#status >|= check_status cmd
+  Lwt_process.with_process_none cmd (fun proc ->
+      proc#status >|= check_status cmd)
```

Comment:
style: I find it more readable to use the `@@` operator in that situation, as you don't need to indent everything afterwards:
```suggestion
  Lwt_process.with_process_none cmd @@ fun proc ->
  proc#status >|= check_status cmd
```